README.md

```diff
@@ -141,6 +149,24 @@ Built-in providers:
 
 Custom providers can be implemented using [Custom Providers](https://strandsagents.com/latest/user-guide/concepts/model-providers/custom_model_provider/)
 
+### Streaming Support
```

Comment:
I feel like this is more docs than needing to be prominent in the readme. I might be alone in that